toonz/sources/tnztools/brushtool.cpp

```diff
@@ -2560,6 +2564,12 @@ void BrushData::saveData(TOStream &os) {
   os.openChild("Modifier_Opacity");
   os << m_modifierOpacity;
   os.closeChild();
+  os.openChild("Modifier_Eraser");
+  os << m_modifierEraser;
+  os.closeChild();
+  os.openChild("Modifier_LockAlpha");
+  os << m_modifierLockAlpha;
```

Comment:
How about modifying the lines 2568 and 2571 as follows?:
```
2568  os << (int)m_modifierEraser;
2571  os << (int)m_modifierLockAlpha;
```

Since AppVeyor claims build errors on Windows as follows:

> C:\projects\opentoonz\toonz\sources\tnztools\brushtool.cpp(2568): error C2666: 'TOStream::operator <<' : 4 overloads have similar conversions [C:\projects\opentoonz\toonz\x64\tnztools\tnztools.vcxproj]
> C:\projects\opentoonz\toonz\sources\tnztools\brushtool.cpp(2571): error C2666: 'TOStream::operator <<' : 4 overloads have similar conversions [C:\projects\opentoonz\toonz\x64\tnztools\tnztools.vcxproj]

Reply:
oops, thank you

i've fixed it now